bin/live/pycbc_live_supervise_collated_triggers

```diff
@@ -0,0 +1,432 @@
+            year=true_utc[0],
+            month=true_utc[1],
+            day=true_utc[2],
+            hour=true_utc[3],
+            minute=true_utc[4],
+            second=true_utc[5]
+        )
+        # Original time of the data being replayed right now
+        current_date = dt_true_start + timedelta(seconds=time_since_replay)
+    else:
+        current_date = day_dt
+
+    date_test = current_date + timedelta(days=1)
+
+    logging.info("Finding fit_by_template files for combination")
+
+    trfits_files = []
+    missed_files = 0
+    found_files = 0
+    while found_files < combined_days and missed_files < 10:
+        # Loop through the possible file locations and see if the file exists
+        date_test -= timedelta(days=1)
+        date_out = date_test.strftime("%Y_%m_%d")
+        fbt_out_fname = option_values['fit-by-template-format'].format(
+            date=date_out,
+            ifo=ifo,
+        )
+        output_dir = os.path.abspath(option_values['output-directory'])
+        fbt_out_full = os.path.join(output_dir, fbt_out_fname)
+        # Check that the file exists:
+        if not os.path.exists(fbt_out_full):
+            missed_files += 1
+            logging.info("File %s does not exist - skipping", fbt_out_full)
+            continue
+        if not len(trfits_files):
+            end_date = date_out
+        # This is now the oldest file
+        first_date = date_out
+        # reset the "missed files" counter, and add to the "found files"
+        missed_files = 0
+        found_files += 1
+        trfits_files.append(fbt_out_full)
+
+    if missed_files == 10:
+        # If more than 10 days between files, something wrong with analysis.
+        # warn and use fewer files - 10 here is chosen to be an unusual amount
+        # of time for the analysis to be down in standard operation
+        logging.warning('More than 10 days between files, only using '
+                        f'{found_files} files for combination!')
+
+    file_id_str = f'{first_date}-{end_date}'
+    out_fname = f'{file_id_str}-{ifo}-FIT_OVER_MULTIPARAM'
+
+    fit_over_args = ['pycbc_fit_sngls_over_multiparam', '--template-fit-file']
+    fit_over_args += trfits_files
+    fit_over_args += sv.dict_to_args(
+        {opt: v for opt, v in option_values.items() if opt in fit_over_multiparam_options}
+    )
+    output_dir = os.path.abspath(os.path.join(
+            option_values['output-directory'],
+            day_str,
+        )
+    )
+    fit_over_full = os.path.join(output_dir, out_fname + '.hdf')
+    fit_over_args += ['--output', fit_over_full]
+    logging.info(fit_over_args)
+    sv.run_and_error(fit_over_args, option_values)
+    if 'variable-fit-over-param' in option_values:
+        variable_fits = option_values['variable-fit-over-param'].format(ifo=ifo)
+        sv.symlink(fit_over_full, variable_fits)
+
+    return fit_over_full, file_id_str
+
+
+def do_collation_fits_dq(args, option_values, day_dt, day_str):
```

Comment:
why does the name of this function include `_dq` ? Is the intention that I add the dq fits into this method after this is merged?

Reply:
That was the plan. The name was intended to be: Do the collation, do the fits, and do the dq